Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM openjdk:8-jre-slim
 LABEL maintainer="cgiraldo@gradiant.org" \
       organization="gradiant.org"
 
-ARG version=2.7.7
+ARG version=3.1.3
```

Comment:
We should keep `version` build arg, since we use lowercase by agreement in Gradiant and `version` is a common build arg for multiple Dockerfiles at Gradiant.